### ISSUE TITLE
Lighthouse SEO & Accessibility are not captured for Desktop version 

### ISSUE THREAD

**Issue by prabuvjy4** [GoogleChrome/lighthouse#12802]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.neimanmarcus.com/?PERF_LOG_LEVEL=5&ENABLE_AUTOPLAY=false

### What happened?

We are running the Lighthouse Audit report through the speed curve tool. Starting from July 15th we are unable to get the Lighthouse SEO and Accessibility score for desktop alone.

Getting Error : Required Accessibility gatherer encountered an error: Cannot read property 'level' of undefined

### What did you expect?

Will able to get the Lighthouse SEO and Accessibility metrics

### What have you tried?

 I have tried running the lighthouse audit through normal chrome browser and public web page test tool without any issues. 

### How were you running Lighthouse?

WebPageTest

### Lighthouse Version

Lighthouse6

### Chrome Version

Version 91.0.4472.124

**Comment by adamraine** [comment]
I can't reproduce this directly on the latest version of LH or v6.x.x. Opening a discussion at https://forums.webpagetest.org/ might be the best bet.

**Comment by patrickhulce** [comment]
We can't reproduce this issue with latest Lighthouse @prabuvjy4, sorry! If you're able to find a minimal repro that fails the accessibility category or the axe extension. I'm sure they'd love an issue over at https://github.com/dequelabs/axe-core :)
